e2e/testsuite/testsuite.go

```diff
@@ -135,13 +135,6 @@ func (s *E2ETestSuite) ConfigureRelayer(ctx context.Context, chainA, chainB ibc.
 	s.startRelayerFn = func(relayer ibc.Relayer) {
 		err := relayer.StartRelayer(ctx, eRep, pathName)
 		s.Require().NoError(err, fmt.Sprintf("failed to start relayer: %s", err))
-		s.T().Cleanup(func() {
-			if !s.T().Failed() {
-				if err := relayer.StopRelayer(ctx, eRep); err != nil {
-					s.T().Logf("error stopping relayer: %v", err)
-				}
-			}
-		})
```

Comment:
this block isn't required as the containers will be cleaned up at the end of the tests anyway. This was also removing the hermes container before the logs would be collected.